sidebars.js

```diff
@@ -1,4 +1,151 @@
 module.exports = {
+  docs: [
+    {
+      type: 'category',
+      label: 'Introduction',
+      items: [
+        'introduction/About-NeoAlgo',
+        'introduction/development-setup',
+        'introduction/contributor-handbook',
+      ]
+    },
+    {
+      type: 'category',
+      label: 'Code Playgrounds',
+      items: [
+        'code-playgrounds/python-playground',
+        'code-playgrounds/java-playground',
+      ]
+    },
+    {
+      type: 'category',
+      label: 'Searching Algorithms',
+      items: [
+        'searching-algorithms/searching-algorithms',
+        'searching-algorithms/linear-search',
+        'searching-algorithms/interpolation-search'
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Sorting Algorithms',
+      items: [
+        'sorting-algorithms/sorting-algorithms',
+        'sorting-algorithms/bubble-sort'
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Data Structures',
+      items: [
+        'data-structures/data-structures',
+        'data-structures/binary-tree'
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Dynamic Programming',
+      items: [
+        'dynamic-programming/dynamic-programming',
+        'dynamic-programming/egg-dropping-problem'
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Computational Geometry',
+      items: [
+        'computational-geometry/computational-geometry',
+        'computational-geometry/jarvis-algorithm'
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Mathematics',
+      items: [
+        'mathematics/mathematics',
+        'mathematics/armstrong-number',
+        'mathematics/factorial-of-number',
+        'mathematics/prime-number',
```

Comment:
You seem to have made an error somewhere. The Prime Number page is not visible on the sidebar. Can you test locally?

Reply:
@HarshCasper every time a new PR is merged so it is showing conflicts and hence not working.

Reply:
Can you please resolve the merge conflicts and see if it works fine then?